src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ CHANGELOG
  * Added `LoginThrottlingListener`.
  * Added `LoginLinkAuthenticator`.
  * Moved methods `supports()` and `authenticate()` from `AbstractListener` to `FirewallListenerInterface`.
+ * [BC break] `$passwordUpgrader` in `PasswordUpgradeBadge` is now nullable (in 2nd argument and `getPasswordUpgrader()`)
```

Comment:
I don't think it's a BC break actually as that constructor arg still accepts the same type, so we just allow omitting it or using null.

Reply:
Yes, but the fact that `getPasswordUpgrader()` might return `null` now actually is a small BC break because code calling that method might not expect that.

Reply:
Indeed I missed the `getPasswordUpgrader()` part, I'd reword the message to talk only about it then.